File: GamesaveCloudLib/SaveGame.cs

namespace GamesaveCloudLib
{
    public class SaveGame
    {
        public int game_id;
        public int savegame_id;
        public string title;
        public string path;
        public Google.Apis.Drive.v3.Data.File driveFolder;

        public SaveGame(int game_id, int savegame_id, string title, string path, Google.Apis.Drive.v3.Data.File driveFolder)
        {
            this.game_id = game_id;
            this.savegame_id = savegame_id;
            this.title = title;
            this.path = path;
            this.driveFolder = driveFolder;
        }
    }
}

File: GamesaveCloudLib/SaveGame.vb
Imports Google.Apis.Drive.v3
Public Class SaveGame
    Public game_id As Integer
    Public savegame_id As Integer
    Public title As String
    Public path As String
    Public driveFolder As Data.File

    Public Sub New(game_id As Integer, savegame_id As Integer, title As String, path As String, driveFolder As Data.File)
        Me.game_id = game_id
        Me.savegame_id = savegame_id
        Me.title = title
        Me.path = path
        Me.driveFolder = driveFolder
    End Sub
End Class
